app/api/voicevox/route.ts

```diff
@@ -0,0 +1,74 @@
+import { NextResponse, NextRequest } from "next/server";
+
+export const GET = async (request: NextRequest) => {
+  // テキストデータを取得
+  const voiceText: string | null = request.nextUrl.searchParams.get("text");
+  // 環境変数からAPIキーを取得
+  const apiKey: string = process.env.VOICEVOX_API_KEY || "";
+
+  // 空白チェック
+  if (!voiceText) {
+    return new NextResponse(JSON.stringify("text parameter is required"), {
+      status: 400,
+    });
```

Comment:
JSONを返すだけなら専用の関数を使う方が適切です。
```suggestion
    return NextResponse.json({message: "text parameter is required"}, { status: 400 });
```